internal/cmd/plugin/command_install.go

```diff
@@ -125,26 +119,26 @@ func getLanguage(manifest *Manifest) (string, *version.Version) {
 		return "", nil
 	}
 
-	primaryDependency := manifest.Dependencies[0]
-	primaryDependency.Dependency = strings.ToLower(primaryDependency.Dependency)
-	if primaryDependency.Version == "" {
-		return primaryDependency.Dependency, nil
+	language := manifest.Dependencies[0]
+	language.Dependency = strings.ToLower(language.Dependency)
+	if language.Version == "" {
+		return language.Dependency, nil
 	}
 
-	ver, err := version.NewVersion(primaryDependency.Version)
+	ver, err := version.NewVersion(language.Version)
 	if err != nil {
-		return primaryDependency.Dependency, nil
+		return language.Dependency, nil
 	}
 
-	return primaryDependency.Dependency, ver
+	return language.Dependency, ver
 }
 
 func checkPythonVersion(ver *version.Version) {
-	versionCmd := exec.NewCommand(Python, "--version")
+	versionCmd := exec.NewCommand("python", "--version")
```

Comment:
This binary is often called `python3` on some machines, so we will want to support that!